submissions/management/commands/update_submissions_uuids.py

```diff
@@ -24,21 +25,43 @@ class Command(BaseCommand):
     """
     help = 'Loads and saves all Submissions objects to force new non-hyphenated uuid values on disk.'
 
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+
+        Uses argparse syntax.  See documentation at
+        https://docs.python.org/3/library/argparse.html.
+        """
+        parser.add_argument(
+            '--start', '-s',
+            default=0,
+            help=u"0-based index at which to begin the update. Default 0."
```

Comment:
Do you want to mention this is the id of the submissions table here?  Helping other folk understand what to feed it or what to target.